### PR TITLE
avatar: add badge on mini size

### PR DESCRIPTION
**Pull request by danidin**

### CODE REVIEW

File: projects/ui-framework/src/lib/buttons-indicators/avatar/avatar-variants.scss [hunk 1]
```diff
@@ -10,6 +10,12 @@
   &.clickable .avatar:hover {
     box-shadow: 0 3px 6px 0 $boxShadowColor;
   }
+  .avatar-badge {
+    width: 16px;
+    height: 16px;
+    top: -2px;
+    right: -1px;
+  }
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/buttons-indicators/avatar/avatar-variants.scss]: @galloween , please take a look

Comment (galloween) [reply projects/ui-framework/src/lib/buttons-indicators/avatar/avatar-variants.scss]: image

it looks like this. if designers approve it - then be it.

**Comment by danidin** [comment]
> @danidin @omrihecht
> 
> We already had the badge on mini before (then we decided to remove it for whatever reason, so now it will make more sense to put it back exactly like it was before ;)
> 
> Changes requested:
> 
> 1. in avatar.consts.ts, line 6, change to `[AvatarSize.mini]: IconSize.small,`
> 2. in avatar.component.scss, remove all width & height for ALL badge sizes
> 3. in avatar.component.scss, for the avatar-badge position, put `top: 0, right: -1px;`

@galloween  
done

**Comment by galloween** [comment]
> > @danidin @omrihecht
> > We already had the badge on mini before (then we decided to remove it for whatever reason, so now it will make more sense to put it back exactly like it was before ;)
> > Changes requested:
> > 
> > 1. in avatar.consts.ts, line 6, change to `[AvatarSize.mini]: IconSize.small,`
> > 2. in avatar.component.scss, remove all width & height for ALL badge sizes
> > 3. in avatar.component.scss, for the avatar-badge position, put `top: 0, right: -1px;`
> 
> @galloween
> done

sorry,  i meant the avatar-variants.scss file..., so:
1) the `top: 0, right: -1px;` should go to the mini section of the variants file (~ line 16)
2) all the avatar-badge sizes should be removed from the variants file
3) the positions (line 41-42) should be removed from the avatar.component.scss now (because they duplicate the .variants file)
sorry for confusion!
Almost there! ;)
thanks!

**Comment by danidin** [comment]
> > > @danidin @omrihecht
> > > We already had the badge on mini before (then we decided to remove it for whatever reason, so now it will make more sense to put it back exactly like it was before ;)
> > > Changes requested:
> > > 
> > > 1. in avatar.consts.ts, line 6, change to `[AvatarSize.mini]: IconSize.small,`
> > > 2. in avatar.component.scss, remove all width & height for ALL badge sizes
> > > 3. in avatar.component.scss, for the avatar-badge position, put `top: 0, right: -1px;`
> > 
> > 
> > @galloween
> > done
> 
> @danidin
> sorry, i meant the avatar-variants.scss file..., so:
> 
> 1. the `top: 0, right: -1px;` should go to the mini section of the variants file (~ line 16)
> 2. all the avatar-badge sizes should be removed from the variants file
> 3. the positions (line 41-42) should be removed from the avatar.component.scss now (because they duplicate the .variants file)
>    sorry for confusion!
>    Almost there! ;)
>    thanks!

@galloween 
yeah, makes sense :)
done

**Pull request merged**